app/views/super_admin/users/edit.html.erb

```diff
@@ -35,6 +35,20 @@
             required: true } %>
       </div>
 
+      <% departments = @user.org.departments.order(:name) %>
+      <% if departments.count > 0 %>
+      <div class="form-group col-xs-8">
+        <% dept_id = @user.department.nil? ? -1 : @user.department.id  %>
+        <%= f.label(:department_id, _('Department or school'), class: 'control-label') %>
+        <%= select_tag(:department_id,
+            options_from_collection_for_select(departments, "id", "name", dept_id),
+            include_blank: true,
+            disabled: departments.count === 0,
```

Comment:
`departments.count === 0` should never happen since we have the `if departments.count > 0` above

Reply:
also, consider using `length` or `any?` instead of count. Count is slower. 

It would also be better to load this in the controller and then reference it via `@departments`